libimage/copier.go

```diff
@@ -60,6 +60,13 @@ type CopyOptions struct {
 	CertDirPath string
 	// Force layer compression when copying to a `dir` transport destination.
 	DirForceCompress bool
+
+	// ImageListSelection is one of CopySystemImage, CopyAllImages, or
+	// CopySpecificImages, to control whether, when the source reference is a list,
+	// copy.Image() copies only an image which matches the current runtime
+	// environment, or all images which match the supplied reference, or only
+	// specific images from the source reference.
```

Comment:
(Without exposing also `Instances`, `CopySpecificImages` is not usable. But `Instances` is not relevant to the current use case anyway, and there’s not really a benefit to explicitly rejecting the value — anyone implementing a user of `CopySpecificImages` would see that it’s impossible.)